### ISSUE TITLE
Read Fairness Article

### ISSUE THREAD

**Issue by ajhaller** [sds-capstone/2022-09-proj7-women-at-table#9]

**Comment by rporta23** [comment]
Kypraiou, S. (2021). What is Fairness? . Feminist AI. Retrieved from https://feministai.pubpub.org/pub/what-is-fairness-

**Comment by rporta23** [comment]
# Notes

* Fairness is inherently subjective and contextual, no tool or software can fully "de-bias" data
* <b>impossibility theorem</b>: no more than one of the three fairness metrics (risk assignments) of demographic parity, predictive parity, and equalized odds can hold at the same time for a well calibrated classifier
* <b>Group Fairness Metrics</b>: "groups should receive similar treatments or outcomes" (e.g. males and females)
    * <b>Demographic Parity</b>: acceptance rates for all demographic groups must be equal
        * guideline: four-fifths rule-- hires in one group should be no less than 4/5 of hires of another group
        * benefits: helps to even out imbalances created by historic bias/discrimination
        * drawbacks: may result in hiring less qualified people if there is more training data for majority demographics because the model may not be able to identify qualifications as well in minority applicants, so the process of hiring may be more random
    * <b>Equalized Odds</b>: The probability of a qualified applicant being hired must be the same for all groups
        * benefits: more likely that all or most of hired applicants are qualified
        * drawbacks: may not help to even out numbers as well and may even reinforce historic bias because minority groups may have lower numbers of qualified applicants overall due to historic lack of opportunities, also makes it more difficult for qualified applicants of all groups to get the job
* <b>Individual Fairness Metrics</b>: "similar people should be treated similarly"
    * <b>Generalized Entropy Index</b>: measures how evenly members of each group are distributed within the applications (0 represents perfect equality, higher numbers represent higher levels of inequality)
        * benefits: more fine-grained than group metrics because considers individual differences/similarities within groups; accounts for intersectionality; uses multiple variables to determine "similarity" rather than just one overarching group
        * drawbacks: defining "similarity" is tricky
* There often must be trade-offs between group fairness and individual fairness-- increasing one decreases the other
    * There even will be trade-offs between demographic parity and equalized odds-- although both are group metrics, we cannot have both at once.
* There also often must be trade-offs between fairness and accuracy-- increasing fairness often decreases accuracy, but it is important to think about the real impacts that our models are having on people's lives in addition to the accuracy of the predictions
* TLDR: Fairness is complex, and there is no approach that will work in all cases. It is a continual process dependent on the stakeholders and applications.

**Comment by ajhaller** [comment]
Read

**Comment by MargaretBassney** [comment]
read

**Comment by alejanmg** [comment]
read
